documentation/modules/post/hardware/automotive/can_flood.md

```diff
@@ -0,0 +1,75 @@
+## Introduction
```

Comment:
Please use consistent whitespace around headers.

Reply:
This is not resolved. I will fix it myself.

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/commit/f5f4c4bec2ada532e6a8eaabc1f695fb6fb02bbd.